src/index.html

```diff
@@ -4,9 +4,95 @@
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
     <title>Meals App</title>
   </head>
   <body>
-    <h1>Meals App</h1>
+    <div class="container mt-4 sticky-top">
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand border border-primary border-4 p-2 rounded-pill" href="/">
+            <b><i>Meals App</i></b>
+          </a>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0" id="navArea">
+              <li class="nav-item">
+                <a class="nav-link" aria-current="page" href="#" id="Seafood"
+                  >Seafood</a
+                >
+              </li>
+
+              <li class="nav-item">
+                <a id="Beef" class="nav-link" aria-current="page" href="#"
+                  >Beef</a
+                >
+              </li>
+
+              <li class="nav-item">
+                <a id="Vegetarian" class="nav-link" aria-current="page" href="#"
+                  >Vegetarian</a
+                >
+              </li>
+            </ul>
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <small><i>Comment and like your favorite meal</i></small>
+              </li>
+          </ul>
+          </div>
+        </div>
+      </nav>
+    </div>
+    <hr>
+    <div class="content mt-2 p-3">
+      <div class="row" id="contentArea"></div>
+    </div>
+
+    <div
+      class="d-none modal fade"
+      id="contentModalBody"
+      tabindex="-1"
+      aria-labelledby="modalTitle"
+      aria-hidden="true"
+    >
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title" id="modalTitle"></h5>
+            <button
+              type="button"
+              class="btn-close"
+              data-bs-dismiss="modal"
+              aria-label="Close"
+            ></button>
+          </div>
+            <img id="modalImage" src="#" alt="Empty Image" class="imageModal">
+
+          <div class="modal-body" id="modal-content"></div>
+        </div>
+      </div>
+    </div>
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
   </body>
 </html>
```

Comment:
- Nice we have the header
- Nice we have the main section

However, we are missing the footer section. Kindly add the footer section.